### DISCUSSION TITLE
What fictional society has its headquarters at 12 Grimmauld Place, London?

### DISCUSSION THREAD

**Discussion by ParanoidUser** [ParanoidUser/yolo#888] [Galaxy Trivia] [answered]
**Bonus:** Pin 📌 the actual location address on a map.

**Comment by peterbaccaro** [comment, marked as answer]
The fictional society that has its headquarters at 12 Grimmauld Place in London is the **Order of the Phoenix**. The Order of the Phoenix is an organization in J.K. Rowling's Harry Potter series. It is a secret society founded by Albus Dumbledore to combat Lord Voldemort and his Death Eaters. 12 Grimmauld Place serves as the main base of operations for the Order, and it is a significant location throughout the series. It is also the ancestral home of the Black family, which is inherited by Sirius Black, one of the members of the Order.

On a map of London, 12 Grimmauld Place would be a fictional location and not a real address. In the Harry Potter series, Grimmauld Place is described as a hidden street in the borough of Islington, London - just a short walk up from King's Cross Station (where Platform 9 3/4 is located).  It was inspired by Lincoln's Inn Fields - but Claremont Square in London was actually used as the exterior.

![image](https://github.com/ParanoidUser/yolo/assets/74666079/59b7dd9f-961a-4a6e-be61-671876eb0292)
